### PR TITLE
Add two fonts based on Unicode 13 2x3 TRS-80/Teletext mosaic characters

### PR DESCRIPTION
**Pull request by rbanffy**
##### Checklist
- [X] I've ensured that similar functionality has not already been implemented
- [X] I've ensured that similar functionality has not earlier been proposed and declined
- [X] I've branched off the `master` or `python-dual-support` branch
- [X] I've merged fresh upstream into my branch recently
- [X] I've ran `tox` successfully in local environment
- [x] I've included docstrings and/or documentation and/or examples for my code (if this is a new feature)

##### Description:
Add two fonts, one with small digits with a 3x5 on 4x6 matrix, and another with a 5x7 on 6x9 matrix characters to the font.py file. These characters use recently added Unicode 13 symbols.

The PR has two small interim fixes for character width calculation that allow it to run, but that should be removed before merging (it can't be merged before #225 is fixed).

### CODE REVIEW

File: source/str_util.c [hunk 1]
```diff
@@ -77,6 +77,7 @@ static const int widths[] = {
     65500, 1,
     65510, 2,
     120831, 1,
+    130047, 1,
```
Comment (rbanffy) [inline source/str_util.c:80]: Interim fix

File: urwid/old_str_util.py [hunk 2]
```diff
@@ -72,6 +72,7 @@
     (65500, 1),
     (65510, 2),
     (120831, 1),
+    (130047, 1),
```
Comment (rbanffy) [inline urwid/old_str_util.py:75]: Interim fix

**Comment by rbanffy** [comment]
If a proper fix to #225 takes too long, can we merge this as is?

**Comment by rbanffy** [comment]
Hi @urwid folks. This has been sitting here for quite some time. The characters used are supported directly by terminals based on VTE (Gnome Terminal and others), QtTermWidget (Konsole, etc). Can we merge this?

**Comment by rbanffy** [comment]
Hi, @ulidtko, do you think this is mergeable?

**Review by penguinolog — COMMENTED** [review]
Validated, fonts rendered correctly.
Interim changes is required in current moment.

**Comment by rbanffy** [comment]
> Validated, fonts rendered correctly. Interim changes is required in current moment.

I've bad news. I just did a fresh install and the examples/bigtext.py program that showcases the new fonts is failing with a

```
Traceback (most recent call last):
  File "/tmp/urwid/examples/bigtext.py", line 163, in <module>
    main()
...
  File "/tmp/urwid/urwid/canvas.py", line 402, in __init__
    raise CanvasError(f"Attribute extends beyond text \n{text[i]!r}\n{attr[i]!r}" )
urwid.canvas.CanvasError: Attribute extends beyond text 
b'      '
[(None, 16)]
```

I'll see if I can add a fix and a test to prevent this regression.

**Comment by penguinolog** [comment]
This issue is not produced by changes in this PR (happens on almost all fonts)

**Comment by penguinolog** [comment]
Real issue is caused by encoding to bytes, when equal length Unicode strings became not equal length byte strings and `TextOverlay` validation fails on lines with empty elements (space placeholders using 1 byte)

**Comment by penguinolog** [comment]
@rbanffy Issue, which you hit is: #496

**Comment by andresmrm** [comment]
Are you sure it's related to that issue?
I created this https://github.com/urwid/urwid/issues/554 to help others getting this error.
For me it only happens with BigText and even in cases that don't use ellipsis.

**Pull request merged**